cli/README.md

```diff
@@ -0,0 +1,58 @@
+# Squoosh CLI
+
+Squoosh CLI is an _experimental_ way to run all the codecs you know from the [Squoosh] web app on your command line using WebAssembly. The Squoosh CLI uses a worker pool to parallelize processing images. This way you can apply the same codec to many images at once.
```

Comment:
I don't know how Markdown references work, does `[Squoosh]` work despite reference having `[squoosh]` casing?

Reply:
Yeah that works :)